src/include.am

```diff
@@ -5,6 +5,7 @@ include src/lib/ctime/include.am
 include src/lib/compress/include.am
 include src/lib/container/include.am
 include src/lib/crypt_ops/include.am
+include src/lib/defs/include.am
```

Comment:
What does `defs/` stands for here? "Definitions" I presume? Would we put all global constant in there or ?...

Reply:
yes, definitions.  But I think only top-level definitions that need to be really global.